### PR TITLE
Fix incorrect api url and add omitemtpy for optional params

### PR DESCRIPTION
**Pull request by SoeunSona**
I tested new api locally just in case and found out some bugs. So I fixed it

### CODE REVIEW

File: chartmogul.go [hunk 1]
```diff
@@ -41,7 +41,7 @@ const (
 )
 
 var (
-	url     = "http://localhost:3000/api/v1/%v"
+	url     = "https://api.chartmogul.com/v1/%v"
```
Comment (SoeunSona) [inline chartmogul.go:44]: My bad. I should have checked this url before merging the pr.

File: notes.go [hunk 2]
```diff
@@ -15,11 +15,11 @@ type Note struct {
...
-	Text         string `json:"text"`
-	AuthorEmail  string `json:"author_email"`
-	CallDuration uint32 `json:"call_duration"`
-	CreatedAt    string `json:"created_at"`
-	UpdatedAt    string `json:"updated_at"`
+	Text         string `json:"text,omitempty"`
+	AuthorEmail  string `json:"author_email,omitempty"`
+	CallDuration uint32 `json:"call_duration,omitempty"`
+	CreatedAt    string `json:"created_at,omitempty"`
+	UpdatedAt    string `json:"updated_at,omitempty"`
```
Comment (SoeunSona) [inline notes.go:22]: Because this params are optional for note api, I added omitempty

File: notes.go [hunk 3]
```diff
@@ -29,11 +29,11 @@ type NewNote struct {
...
-	AuthorEmail  string `json:"author_email"`
-	Text         string `json:"text"`
-	CallDuration uint32 `json:"call_duration"`
-	CreatedAt    string `json:"created_at"`
-	UpdatedAt    string `json:"updated_at"`
+	AuthorEmail  string `json:"author_email,omitempty"`
+	Text         string `json:"text,omitempty"`
+	CallDuration uint32 `json:"call_duration,omitempty"`
+	CreatedAt    string `json:"created_at,omitempty"`
+	UpdatedAt    string `json:"updated_at,omitempty"`
```
Comment (SoeunSona) [inline notes.go:36]: Because this params are optional for note api, I added omitempty

**Review by daesu — APPROVED** [review]
lgtm

**Pull request merged**